drv/Function.drv

```diff
@@ -104,9 +159,13 @@ public interface FUNCTION KEY_VALUE_GENERIC extends Function<KEY_GENERIC_CLASS,
 
 	/** {@inheritDoc}
 	 * @deprecated Please use the corresponding type-specific method instead. */
+
 	@Deprecated
 	@Override
-	boolean containsKey(Object key);
+	default boolean containsKey(Object key) {
```

Comment:
No, the containsKey() implementation is just for primitive keys.

#if KEYS_PRIMITIVE
    public boolean containsKey( final Object ok ) {
        if ( ok == null ) return false;
        return containsKey( KEY_OBJ2TYPE( ok ) );
    }
#endif

I don't know what you mean by the put(null, 1) example. This problem is only for primitive keys. When handling primitive keys, you check for null on reading and act as if it wasn't in the map. For the rest, OBJ2TYPE().